src/controllers/crossCutting.controller.js

```diff
@@ -51,12 +53,17 @@ async function crossCutting(req, res, next) {
   }
 
   await exportFinalVideo(TEMP_DIR_FRAMES.MAIN);
-  progressStatus.stage = "completed";
 
+  const s3ClientFinalVideoUrl = await getFinalVideoUrl();
+
+  progressStatus.stage = "completed";
   res.status(200).send({
-    result: "success",
-    message: "final video is exported",
+    lastResult: "success",
+    s3ClientFinalVideoUrl,
```

Comment:
바로 메세지에 완료된 영상의 Client Url 보내주시는군요! 다만 클라이언트 측에서 받아쓰기 용이하도록 프로퍼티값을 주면 좋을 것 같다는 의견입니다! 

```suggestion
    message: s3ClientFinalVideoUrl,
    // 혹은 finalVideo: s3ClientFinalVideoUrl
```

Reply:
리소스 파일 생성에 따라 status code를 세심하게 신경써주셨네요 :) 확인하였습니다!